scripts/communityScripts/chat/FloofChat.js

```diff
@@ -464,19 +470,22 @@ function messageReceived(channel, message) {
 
 function time() {
     var d = new Date();
+    var month = (d.getMonth()).toString();
+    var day = (d.getDate()).toString();
     var h = (d.getHours()).toString();
     var m = (d.getMinutes()).toString();
     var s = (d.getSeconds()).toString();
     var h2 = ("0" + h).slice(-2);
     var m2 = ("0" + m).slice(-2);
     var s2 = ("0" + s).slice(-2);
-    s2 += (d.getMilliseconds() / 1000).toFixed(2).slice(1);
-    return h2 + ":" + m2 + ":" + s2;
+    // s2 += (d.getMilliseconds() / 1000).toFixed(2).slice(1);
+    return month + "/" + day + " - " + h2 + ":" + m2 + ":" + s2;
```

Comment:
Hmmm. It might be nicer to use a user's local date format, e.g., so that there's no ambiguity over what 2/3 means. E.g., see Date.toLocaleDateString().

Reply:
Perhaps but the issue with that is that I have to add detection of the user's country and then figure out what that format will be. It might be easier to just follow ISO standards which is what I think I am doing now? It does it like so: MM/DD

Reply:
Date.toLocaleDateString() should work without needing any detection of user's country etc.

Reply:
You are right, I tried and it worked when I specify 'undefined', however it would not work at all no matter what options I specified. This caused a full date with even the day of the week to be outputted. It took up too much space then, and without a UI library like Vuetify, it'd be tough to find a way to fit in something like that without moving around things.

I figure that https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString was added pretty early, with all parameters being fully implemented by Chrome 24. I do not see why the QtScript engine would not have this implemented properly.